src/App.tsx

```diff
@@ -59,7 +59,7 @@ function App() {
       errorElement: <NotFound />,
       children: [
         { index: true, element: <Dashboard  /> },
-        { path: 'tasksData', element: <TasksData/> },
+        { path: 'tasksList/task-form', element: <TaskForm/> },
```

Comment:
No need for adding `List` at the end of tasks as it can be inferred from the plural noun.